packages/repository/README.md

```diff
@@ -168,7 +168,13 @@ coercion. The following types are supported out of box.
 - ArrayType
 - UnionType
 
-# Use Repository
+## Installation
+
+```
+$ npm install --save @loopback/repository
+```
+
+## Basic use
```

Comment:
IMO, the section "Basic Use" should show the most common and simple example of how to use this module.

I am proposing the following:

 - define a custom Model class
 - create repository using `DefaultCrudRepository`

I think there are few missing pieces, e.g. `DefaultCrudRepository` requires a data-source instance that we need to create somehow. Please talk to @raymondfeng about this and make sure the new code snippet is self-contained (it should have `import` statements for classes used) and works correctly.

You may want to take a look at "Define a repository" section, I think some of the code examples can be leveraged for "Basic Use" section.